contrib/systemd/btrbk.timer.in

```diff
@@ -1,9 +1,10 @@
 [Unit]
 Description=btrbk daily backup
+ConditionPathExists=|@CONFDIR@/btrbk/btrbk.conf
```

Comment:
Is it a good idea to set this in the timer as well? a user might want to override the service with `ExecStart=/bin/btrbk run -c some_other_config`, which would somehow break the timer.

Reply:
Hmm... TBH, I'm not exactly sure how "bad" it is to run the time and let just the service not run because of the conditional.

Probably not at all, and at most it would be "conceptually" ugly, i.e. when the timer is considered to have been fired, but the service actually didn't.<br>
But even this may not be considered that way by systemd experts.

On could argue, that if a user wants to override the `.service` he could as well just also override the `.timer`.

So.. yeah... I have no real strong opinion about this.